media-api/app/controllers/MediaApi.scala

```diff
@@ -306,12 +307,29 @@ object MediaApi extends Controller with ArgoHelpers {
     Link("related-labels", uri)
   }
 
+  private val getSuggestedLabelsLink =
+    Link("suggested-labels", s"$rootUri/suggest/edits/labels")
+
   def suggestMetadataCredit(q: Option[String], size: Option[Int]) = Authenticated.async { request =>
     ElasticSearch
       .completionSuggestion("suggestMetadataCredit", q.getOrElse(""), size.getOrElse(10))
       .map(c => respondCollection(c.results))
   }
 
+  def suggestLabels() = Authenticated {
```

Comment:
This is where I am thinking it should be a subset of super labels. We can then autocomplete the [preset labels](https://github.com/guardian/grid/blob/master/kahuna/public/js/services/preset-label.js) too